modules/administration-guide/partials/proc_enabling-readiness-init-containers-for-the-olm-installer.adoc

```diff
@@ -0,0 +1,44 @@
+:_module-type: PROCEDURE
+
+[id="enabling-readiness-init-containers-for-the-olm-installer_{context}"]
+= Enabling the readiness init containers for the OLM installer
+
+The readiness init containers are not enabled by default, so to use them you first have to enable them. To enable the readiness init containers for {prod-short} installed by the OLM installer:
+
+.Prerequisites
+
+* {prod-short} is installed by the OLM installer.
+
+.Procedure
+
+. Find the {prod-short} Operator subscription name:
++
+[source,shell,subs="+quotes,+attributes"]
+----
+$ {orch-cli} get subscriptions -n {prod-namespace}
+----
+
+. Get the CSV (Cluster Service Version) name from the {prod-short} Operator subscription:
```

Comment:
```suggestion
. Find the CSV (Cluster Service Version) name from the {prod-short} Operator subscription:
```

Reply:
Just adding the above suggestion in case you think we'd better use `find` instead of `get` in this particular step.

Reply:
Here we obtain CSV by the command